packages/communicator-bcrypt/package.json

```diff
@@ -23,7 +23,9 @@
     "typescript": "^5.3.3"
   },
   "dependencies": {
-    "@dbos-inc/dbos-sdk": "^0.8.41-preview",
     "bcryptjs": "^2.4.3"
+  },
+  "peerDependencies": {
+    "@dbos-inc/dbos-sdk": "^0.8.51-preview"
```

Comment:
IIUC when one installs this package, it will find and use an existing `@dbos-inc/dbos-sdk`, rather than installing it as a dependency of itself.

The existing packaging would result, after an install, in `node_modules/@dbos-sdk` and `node_modules/dbos-inc/communicator-bcrypt/node_modules/@dbos-inc/dbos-sdk`

Reply:
Right, we don't want the existing way, which produces (a possibly incompatible) `node_modules/dbos-inc/communicator-bcrypt/node_modules/@dbos-inc/dbos-sdk`